Source/WebCore/html/track/VTTCue.cpp

```diff
@@ -1349,6 +1347,38 @@ void VTTCue::toJSON(JSON::Object& object) const
     object.setString("align"_s, align());
 }
 
+void VTTCue::speak(double rate, double volume, SpeakCueCompletionHandler&& completion)
+{
+#if ENABLE(SPEECH_SYNTHESIS)
+    ASSERT(rate);
+    ASSERT(track());
+    if (m_content.isEmpty() || !track())
+        return;
+
+    auto& track = *this->track();
+    if (!m_speechUtterance) {
```

Comment:
Is it possible for `m_speechUtterance` to exist at this point? If so, the completion handler won't be called.

Reply:
It shouldn't be possible, I'll add an assert and will remove the test.